WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -135,6 +135,36 @@ public MainViewModel()
 		FullScreen.CurrentPage is { IsBusy: true } ||
 		CompactDialogScreen.CurrentPage is { IsBusy: true };
 
+	public bool IsDialogOpen()
+	{
+		return DialogScreen.IsDialogOpen
+		       || FullScreen.IsDialogOpen
+		       || CompactDialogScreen.IsDialogOpen;
+	}
+
+	public void ShowDialogAlert()
+	{
+		if (CompactDialogScreen.IsDialogOpen)
+		{
+			CompactDialogScreen.ShowAlert = false;
+			CompactDialogScreen.ShowAlert = true;
+			return;
+		}
+
+		if (DialogScreen.IsDialogOpen)
+		{
+			DialogScreen.ShowAlert = false;
+			DialogScreen.ShowAlert = true;
+			return;
+		}
+
+		if (FullScreen.IsDialogOpen)
+		{
+			DialogScreen.ShowAlert = false;
```

Comment:
Copy paste mistake, It should be `FullScreen`-

Reply:
BTW does the full screen even make any sense? Does the warning even animation visible when it is full screen?

Reply:
Fixed. 

Animation is visible in Add Wallet screen.